pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/ModularLoadManagerStrategy.java

```diff
@@ -55,11 +58,17 @@ Optional<String> selectBroker(Set<String> candidates, BundleData bundleToAssign,
      */
     static ModularLoadManagerStrategy create(final ServiceConfiguration conf) {
         try {
-            // Only one strategy at the moment.
-            return new LeastLongTermMessageRate(conf);
+            Class<?> loadManagerStrategyClass = Class.forName(conf.getLoadBalancerLoadManagerStrategy());
+            Object loadManagerStrategyInstance = loadManagerStrategyClass.newInstance();
```

Comment:
```suggestion
            Object loadManagerStrategyInstance = loadManagerStrategyClass.getDeclaredConstructor().newInstance();
```
`Class#newInstance()` is deprecated.

Reply:
Updated.